src/js/components/user/info.tsx

```diff
@@ -0,0 +1,196 @@
+        {i18n.t('Connected to Salesforce')}
+      </span>
+      {!user.is_devhub_enabled && (
+        <p className="slds-text-body_small slds-m-top_x-small">
+          <Icon
+            assistiveText={{ label: i18n.t('Error') }}
+            category="utility"
+            name="error"
+            colorVariant="error"
+            size="x-small"
+            className="slds-m-bottom_xxx-small"
+            containerClassName="slds-m-right_xx-small"
+          />
+          <Trans i18nKey="devHubNotEnabled">
+            This Salesforce org does not have Dev Hub enabled, and will not be
+            able to create new scratch orgs. Learn how to{' '}
+            <ExternalLink url="https://help.salesforce.com/articleView?id=sfdx_setup_enable_devhub.htm&type=0">
+              enable Dev Hub
+            </ExternalLink>
+            .
+          </Trans>
+        </p>
+      )}
+      <ul className="slds-m-top_small">
+        <li>
+          <strong>{i18n.t('Dev Hub')}:</strong>{' '}
+          {user.is_devhub_enabled ? (
+            <span className="slds-text-color_success">{i18n.t('Enabled')}</span>
+          ) : (
+            <>
+              <span className="slds-text-color_error">
+                {i18n.t('Not Enabled')}
+              </span>
+              {' | '}
+              <Button
+                label={i18n.t('Check Again')}
+                variant="link"
+                onClick={doRefreshDevHubStatus}
+              />
+            </>
+          )}
+        </li>
+        {user.sf_nickname && (
+          <li>
+            <strong>{i18n.t('User')}:</strong> {user.sf_nickname}
+          </li>
+        )}
+        {user.org_name && (
+          <li>
+            <strong>{i18n.t('Org')}:</strong> {user.org_name}
+          </li>
+        )}
+        {user.org_type && (
+          <li>
+            <strong>{i18n.t('Type')}:</strong> {user.org_type}
+          </li>
+        )}
+      </ul>
+      <Button
+        label={i18n.t('Disconnect from Salesforce')}
+        variant="link"
+        className="slds-m-top_small"
+        onClick={doDisconnect}
+      />
+    </>
+  );
+};
+
+const UserInfo = () => {
+  const user = useSelector(selectUserState);
+  const [modalOpen, setModalOpen] = useState(false);
+
+  return user ? (
+    <>
+      <Popover
+        align="bottom right"
+        heading={
+          <div className="slds-p-around_small">
+            <Icon category="utility" name="user" size="small" />
+            <span className="slds-p-left_small">{user.username}</span>
+            <div className="slds-m-left_large slds-p-left_small">
+              <Logout className="slds-text-body_regular" />
+            </div>
+          </div>
+        }
+        body={
+          <div className="slds-p-around_small">
+            {user.valid_token_for ? (
+              <ConnectionInfo user={user} />
+            ) : (
+              <ConnectToSalesforce toggleModal={setModalOpen} />
```

Comment:
@jgerigmeyer I want to make sure I'm not missing something.  When I connect to sf, it doesn't change to show that it's connected :thinking: 
![Screenshot from 2019-08-15 11-23-01](https://user-images.githubusercontent.com/12376362/63105567-16f53100-bf4f-11e9-9726-5751a6df7b51.png)

Reply:
That... is strange. 🤔 I assume you have the latest pulled, etc?

I wonder if we're suppressing an error message somewhere. If you navigate to `/admin/` do you see any helpful messages? Do you have the socialaccounts created etc?

Reply:
ah, yes it does.  There's an error there that reads: `The social account is already connected to a different account.
`

Reply:
Ah, I ran into that earlier, but Kit fixed it late yesterday. Delete your existing SF socialaccounts from the admin, and then try again.

Reply:
@wlonk This does imply that we're not exposing (or even logging) connection errors in a helpful way. Added a story to address that: https://trello.com/c/e1dkbOKV/73-if-there-was-an-error-connecting-to-a-sf-org-i-want-it-to-be-exposed-to-the-user-with-helpful-instructions

Reply:
@duggiemitchell Did changing your permissions on the SF org resolve this issue for you?